nipype/interfaces/mrtrix3/preprocess.py

```diff
@@ -16,87 +16,47 @@
 import os.path as op
 
 from ..base import (CommandLineInputSpec, CommandLine, traits, TraitedSpec,
-                    File, isdefined)
+                    File, isdefined, Undefined)
 from .base import MRTrix3BaseInputSpec, MRTrix3Base
 
 
 class ResponseSDInputSpec(MRTrix3BaseInputSpec):
```

Comment:
Yes, they do since the current version of MRtrix provides different APIs from the ones implemented at the moment in the Nipype interface (https://github.com/nipy/nipype/pull/2299).